server/api/board/[id]/index.get.ts

```
@@ -0,0 +1,457 @@
+      // These require specific access rights
+      if (!profileId) {
+        return false; // Anonymous users cannot access private boards
+      }
+
+      // Check if the user has been granted access
+      const accessRecord = await db.query.BOARD_ACCESS.findFirst({
+        where: and(
+          eq(BOARD_ACCESS.board_id, board.board_id),
+          eq(BOARD_ACCESS.profile_id, profileId),
+        ),
+      });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)